cMake/FreeCAD_Helpers/SetupQt.cmake

```diff
@@ -23,9 +23,9 @@ if(BUILD_GUI)
         elseif(${FREECAD_USE_QTWEBMODULE} MATCHES "Qt WebEngine")
             find_package(Qt5WebEngineWidgets REQUIRED)
         else() # Automatic
-            find_package(Qt5WebKitWidgets QUIET)
+            find_package(Qt5WebEngineWidgets REQUIRED)
```

Comment:
Aren't the REQUIRED and QUIET backward now?

Reply:
Yeah... And that happened because I am totally idiot!